### PR TITLE
Glimpse phase nftest

### PR DESCRIPTION
**Pull request by LouisLeNezet**
## PR checklist

Add nf-test for glimpse_phase

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/glimpse/phase/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,67 @@
+nextflow_process {
+
+    name "Test Process GLIMPSE_PHASE"
+    script "modules/nf-core/glimpse/phase/main.nf"
```
Comment (sateeshperi) [inline modules/nf-core/glimpse/phase/tests/main.nf.test:4]:
```suggestion
    script "../main.nf"
```

**Review by sateeshperi — CHANGES_REQUESTED** [review]
minor change

File: modules/nf-core/glimpse/phase/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        }
+
+        then {
+            String targetFileName = "input_present_chr21_16650000-16800000.vcf.gz"
+            File selectedFile = process.out.phased_variant.stream()
+                .filter(vector -> vector.size() > 1)
+                .map(vector -> new File(vector.get(1).toString()))
+                .filter(file -> file.getName().equals(targetFileName))
+                .findFirst()
+                .orElse(null)
```
Comment (Emiller88) [inline modules/nf-core/glimpse/phase/tests/main.nf.test:55]: Love it!

**Review by Emiller88 — APPROVED** [review]
🤠

**Pull request merged**